notifications_utils/recipient_validation/phone_number.py

```diff
@@ -251,22 +135,28 @@ def parse_phone_number(self, phone_number: str) -> phonenumbers.PhoneNumber:
 
         number = self._try_parse_number(phone_number)
 
-        if str(number.country_code) not in COUNTRY_PREFIXES + ["+44"]:
+        if str(number.country_code) not in COUNTRY_PREFIXES + ["44"]:
             raise InvalidPhoneError(code=InvalidPhoneError.Codes.UNSUPPORTED_COUNTRY_CODE)
 
         if (reason := phonenumbers.is_possible_number_with_reason(number)) != phonenumbers.ValidationResult.IS_POSSIBLE:
             if forced_international_number := self._validate_forced_international_number(phone_number):
                 number = forced_international_number
             else:
                 raise InvalidPhoneError.from_phonenumbers_validation_result(reason)
+
         if not (phonenumbers.is_valid_number(number) & self._is_allowed_phone_number_type(number)):
             # is_possible just checks the length of a number for that country/region. is_valid checks if it's
             # a valid sequence of numbers. This doesn't cover "is this number registered to an MNO".
             # For example UK numbers cannot start "06" as that hasn't been assigned to a purpose by ofcom
             if self._is_tv_number(number):
                 return number
             else:
-                raise InvalidPhoneError(code=InvalidPhoneError.Codes.INVALID_NUMBER)
+                if phonenumbers.number_type(number) == phonenumbers.PhoneNumberType.UNKNOWN:
```

Comment:
All those three cases result in the same error raised - do they need to be three separate cases?

Reply:
This was just some code I needed to clean up when I was trying to debug an issue. It should all go on one line.